### PR TITLE
Fixes to a7 spec

### PR DESCRIPTION
**Pull request by 64bitpandas**
- change /opt/lab7/readonly,readwrite to mnt
 - Piazza delete
 - DNS: instruct students to intentionally bring down bind9
 - Change location of bind9.service to /lib/systemd/system/named.service
 - Link to named.conf.local  in decal-labs
 - Instructions to install HAProxy
 - Install tornado using apt install python3-tornado

### CODE REVIEW

**Review by BernardZhao — APPROVED** [review]
Looks like it fixes everything we talked about before. 👍 

Only thing I'm concerned about is that `staff4.decal.xcf.sh:/opt/lab7/public` will probably have to be changed again in the future. But not a huge problem

File: labs/a7.md [hunk 1]
```diff
@@ -109,16 +109,19 @@ To submit the lab, answer the questions on Gradescope.
...
-not recognize a command) to mount from `staff:/opt/lab7/public` (the remote
-directory) to your local directory at `/opt/lab7/read-only`. Once you do this,
+We have provided a NFS server for you to connect to at `staff4.decal.xcf.sh` with two
+different directories, one read-only and one read-write. 
+
+First, **install the `nfs-common` package** so that you can mount directories over NFS. 
+Then, **use the `mount` command** (remember to look at the `man` pages or search online if you do
+not recognize a command) to mount from `staff4.decal.xcf.sh:/opt/lab7/public` (the remote
+directory) to your local directory at `/mnt/`. Once you do this,
 you should see a file with a secret inside it in `/opt/lab7/read-only`. You can
```
Comment (jvperrin) [inline labs/a7.md:119]: This path needs changing too, it'll be within `/mnt` instead, same with the path on line 133.

File: labs/a7.md [hunk 2]
```diff
@@ -131,14 +134,17 @@ loaded and you will get an error message like `umount.nfs4:
 NFS. If you run `df`, you should see that the entry that was present before has
 now disappeared.
 
-Next, mount the directory at `staff:/opt/lab7/private/<your username>` to
-`/opt/lab7/read-write` using `mount` in a similar way to before. What do you
-see in `/opt/lab7/read-write` now? Follow the instructions in the file given
-there, note that you will have to use `sudo` here too to create a new file
+Now, let's unmount the filesystem using `umount /mnt/` so that we can mount
```
Comment (jvperrin) [inline labs/a7.md:137]: This might give away question 1e, but probably not a big deal

**Review by nikhiljha — APPROVED** [review]

File: labs/a7.md [hunk 3]
```diff
@@ -166,13 +172,16 @@ more information.
 
 [lab-a5]: /labs/a5#dns-configuration
```
Comment (jvperrin) [inline labs/a7.md:173]: I don't actually see a description of DNS in a5 any more, so probably should remove this

File: labs/a7.md [hunk 4]
```diff
@@ -231,9 +240,14 @@ Then, create a file `/etc/bind/db.example.com` to contain the responses to give
...
 there to get what you want for your config instead of having to start from
-scratch. To make this easier, we've provided a valid config at
-`/opt/lab7/db.example.com` that you can copy in place at
-`/etc/bind/db.example.com`. It is prefilled with your VM's IP, and includes a
+scratch. 
+
+To make this easier, we've provided a valid config in 
+[decal-labs](https://github.com/0xcf/decal-labs/blob/master/a7/db.example.com)
+that you can copy in place at
+`/etc/bind/db.example.com`. **You'll need to edit the config to include your VM's IP address!**
```
Comment (jvperrin) [inline labs/a7.md:248]: There's a couple things to edit, one with the IP, and another with a domain name

File: labs/a7.md [hunk 5]
```diff
@@ -269,9 +280,13 @@ commonly-used open-source load balancer. [NGINX](https://nginx.org/) is
 actually [starting to become a load balancer][nginx-lb] alongside being a web
 server, which is pretty interesting, but HAProxy is still commonly used.
 
+You can install HAProxy using `apt install haproxy`.
+
 First, grab the python file for the service you will be running from the
 [decal-labs repo][decal-labs-a7] using `wget` or something similar to download
-it. When run (`python3 server.py`), this script will start up 6 different HTTP
+it. You'll likely also need to install `tornado` using `apt install python3-tornado`.
```
Comment (jvperrin) [inline labs/a7.md:287]: Could add a `sudo` in front of apt if you want to be more precise

**Review by jvperrin — COMMENTED** [review]
Looks pretty good, I have a few comments but the updates so far are great

**Pull request merged**